sorc/build_all.sh

```diff
@@ -116,7 +117,9 @@ build_opts=(
     ["upp"]="${_build_debug}"
     ["ww3_gfs"]="${_verbose_opt} ${_build_debug}"
     ["ww3_gefs"]="-w ${_verbose_opt} ${_build_debug}"
-    ["gdas"]="${_verbose_opt} ${_build_debug}"
+    ["gdas"]="${_verbose_opt} ${_build_debug} ${_workflow_dir}"
```

Comment:
```suggestion
    ["gdas"]="${_verbose_opt} ${_build_debug}"
```
These are unnecessary complications since the path of this can be determined inside the script.